mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ConvertScreen.tsx

```diff
@@ -184,112 +173,204 @@ export function ConvertScreen(props: Props): JSX.Element {
+          }}
+          amountButtonsStyle={{
+            light: tailwind("bg-mono-light-v2-00"),
+            dark: tailwind("bg-mono-dark-v2-00"),
+            style: tailwind("mt-6 rounded-xl-v2"),
+          }}
+        >
+          <View
+            style={tailwind("flex flex-row justify-between items-center pl-5")}
           >
-            <WalletTransactionCardTextInput
-              inputType="numeric"
-              displayClearButton={amount !== ""}
-              displayFocusStyle
-              onChangeText={setAmount}
-              value={amount}
-              hasBottomSheet={false}
-              onFocus={() => setIsInputFocus(true)}
-              onBlur={() => setIsInputFocus(false)}
-              onClearButtonPress={() => setAmount("")}
-              inputContainerStyle={tailwind("pb-5 pt-3 px-0")}
-              placeholder="0.00"
-              testID="convert_input"
-            />
-          </TransactionCard>
-          <View style={tailwind("flex-row items-center px-5 pt-2")}>
-            <ThemedTextV2
-              style={tailwind("font-normal-v2 text-xs")}
-              light={tailwind("text-mono-light-v2-500", {
-                "text-red-v2": inlineTextStatus === InlineTextStatus.Error,
-                "text-orange-v2": inlineTextStatus === InlineTextStatus.Warning,
-              })}
-              dark={tailwind("text-mono-dark-v2-500", {
-                "text-red-v2": inlineTextStatus === InlineTextStatus.Error,
-                "text-orange-v2": inlineTextStatus === InlineTextStatus.Warning,
-              })}
-              testID="source_balance_label"
-            >
-              {translate(
-                "screens/ConvertScreen",
-                inlineTextStatus === InlineTextStatus.Error
-                  ? "Insufficient balance"
-                  : inlineTextStatus === InlineTextStatus.Warning
-                  ? "A small amount of UTXO is reserved for fees"
-                  : "Available: ",
-                {
-                  amount: new BigNumber(sourceToken.amount).toFixed(8),
-                  unit: getDisplayUnit(sourceToken.unit),
-                }
-              )}
-            </ThemedTextV2>
-            {inlineTextStatus === InlineTextStatus.Default && (
+            <View style={tailwind("w-6/12 mr-2")}>
+              <ThemedTextInputV2
+                style={tailwind("text-xl font-semibold-v2 w-full")}
+                light={tailwind("text-mono-light-v2-900")}
+                dark={tailwind("text-mono-dark-v2-900")}
+                keyboardType="numeric"
+                value={amount}
+                onChangeText={setAmount}
+                placeholder="0.00"
+                placeholderTextColor={getColor(
+                  isLight ? "mono-light-v2-900" : "mono-dark-v2-900"
+                )}
+                testID="convert_input"
+              />
               <NumberFormat
+                value={getNumberFormatValue(amount, 2)}
```

Comment:
why are we displaying amount as the `$` value?

Reply:
updated to use getTokenPrice to get the $value of token